pkg/controller/plan/kubevirt.go

```diff
@@ -1311,10 +1311,17 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	if err != nil {
 		return
 	}
+	// pod annotations
+	annotations := map[string]string{}
+	if r.Plan.Spec.TransferNetwork != nil {
+		annotations[AnnDefaultNetwork] = path.Join(
+			r.Plan.Spec.TransferNetwork.Namespace, r.Plan.Spec.TransferNetwork.Name)
+	}
```

Comment:
`if el9`? or we don't care if we have this annotation on both cases(cold+warm)?

Reply:
> `if el9`? or we don't care if we have this annotation on both cases(cold+warm)?

[This patch adds the transfer network, if set, on the conversion pod in both cases.](https://github.com/kubev2v/forklift/pull/673#issue-2028421279) ;)
yeah, the network doesn't matter when virt-v2v doesn't go outside of the pod